dashboard/db/migrate/20160219021859_add_submitted_to_user_levels.rb

```diff
@@ -0,0 +1,14 @@
+class AddSubmittedToUserLevels < ActiveRecord::Migration
+  def change
+    add_column :user_levels, :submitted, :boolean
+
+    reversible do |change|
+      change.up do
+        execute('UPDATE user_levels SET submitted = true, best_result = 30 where best_result = 1000')
+      end
+      change.down do
+        execute('UPDATE user_levels SET best_result = 1000 where best_result = 30')
```

Comment:
Doesn't this convert all existing best_results of 30 to 1000? Don't you need to only do this if submitted = true?